testing/benchmark/bin/parse_and_send.dart

```diff
@@ -38,18 +38,32 @@ Future<List<FlutterEngineMetricPoint>> _parse(String jsonFileName) async {
   return points;
 }
 
+Future<FlutterDestination> connectFlutterDestination() async {
+  const String kTokenPath = 'TOKEN_PATH';
+  const String kGcpProject = 'GCP_PROJECT';
+  final Map<String, String> env = Platform.environment;
+  if (env.containsKey(kTokenPath) && env.containsKey(kGcpProject)) {
+    return FlutterDestination.makeFromAccessToken(
+      env[kTokenPath],
+      env[kGcpProject],
```

Comment:
Technically, no as we default the project id to be `flutter-cirrus` if it's not provided.

However, to avoid any confusion, I felt it's better to set the token and GCP project id in one place because one might need to change the token when the project is changed (or vice versa). The recipe example you gave me also set the token and the project id in one place.

I'm Ok with either way. If there's a difficulty in giving the GCP project id in the environment variables, or if you'd like to omit it, I can easily remove it from this PR.

Reply:
(The current PR does require `env.containsKey(kGcpProject)` to be true though. I can remove it so it's not needed.)

Reply:
I advanced a little bit but the authenticated client is rejecting the token: https://chromium-swarm.appspot.com/task?id=4f8147477ee94610

Reply:
I think I need to add LUCI's service account to flutter-cirrus project's access list. Do you have the id of that service account, or can you put a link to your recipe so maybe I can look it up?

Reply:
The service account is "flutter-try-builder@chops-service-accounts.iam.gserviceaccount.com" I tried adding it yesterday but maybe I didn't granted it the correct permissions.

Reply:
That's strange. "Cloud Datastore User" should be the correct role... Is there someway to verify that the generated token belongs to that service account (e.g. write a script that takes an access token and use some APIs to print out the id)?

Reply:
The easiest way will be to generate a token using ```gcloud auth application-default print-acc
ess-token``` and passing it directly to the tool to see if the tool is processing the token in the expected way.

Reply:
`gcloud auth application-default print-access-token` seems to require a credentials json of the service account to print the access token? If so, I've already done a similar test by generating the token from the credentials and send it to the script: https://github.com/flutter/engine/pull/22019/files#diff-f2e2f624ef84bb33cc75f9c5a09e1c0f9371eb06d5b73ce94fd65ca6ee052b92R40